test/application/support/env.js

```diff
@@ -0,0 +1,26 @@
+const { setWorldConstructor } = require("@cucumber/cucumber");
+
+require("playwright");
+
+const users = {
+  "Authenticatable Anita": {
```

Comment:
I don't think you need these authentication users, but you might want a Not Found Nigel or a Postcode Mismatch Mary in the future when you implement various error paths.

Again, for now, it's syntactic sugar to make it easier to remember what the result should be, as opposed to a generic "I" or "They"